breeze

```diff
@@ -804,14 +804,20 @@ function parse_arguments() {
           COMMAND_TO_RUN="cleanup_image"
           shift ;;
         docker-compose)
-          if [[ "$#" -lt 2 ]]; then
+          LAST_SUBCOMMAND="${1}"
+          if [[ $# -lt 2 ]]; then
             echo "You should specify docker compose command to run"
-            exit 1
+            shift
+            RUN_HELP="true"
+          else
+            DOCKER_COMPOSE_COMMAND="${2}"
+            if [[ ${DOCKER_COMPOSE_COMMAND} == "--help" ]]; then
+                RUN_HELP="true"
+            fi
```

Comment:
I test in local and seem delete this three code and `--help` option still work, also the short option `-h`

Reply:
Right. I forgot that the --options are parsed before and removed :). Good catch. Fixed.

Reply:
I actually found that there was also one thing missing (specification of the command parameter for parameterized command). I added it.